source/SAMRAI/geom/CartesianSideDoubleWeightedAverage.C

```diff
@@ -232,7 +386,9 @@ CartesianSideDoubleWeightedAverage::coarsen(
                cgeom->getDx(),
                fdata->getPointer(2, d),
                cdata->getPointer(2, d));
+               exit(-1);
          }
```

Comment:
My guess is you have these exits to signal where you haven't completed things yet, but these will need to be removed.